projects/novo-elements/src/elements/data-table/data-table.component.scss

```diff
@@ -68,7 +68,8 @@ cdk-table {
 .novo-data-table-header-row {
   display: flex;
   flex-direction: row;
-  flex-wrap: no-wrap;
+  flex-wrap: nowrap;
+  background-color: $white;
   .novo-data-table-header-cell,
```

Comment:
setting `background-color` here would affect every implementation of the data-table. any reason why this style cannot be applied directly within your implementation?

Reply:
For all of the rest of the cells and rows in this file, we specifically hard code the colors.  Novo-Data-Tables has not been something that is customized in any of our implementations.  Normally it has a background of white, but when you scroll, you find out it actually has a background that is transparent @amrutha-rajiv